third_party/icu/scripts/make_data_assembly.py

```diff
@@ -42,11 +42,11 @@
   step = -1
 
 input_data = open(input_file, 'rb').read()
-n = input_data.find("icudt")
+n = input_data.find(b'icudt')
 if n == -1:
   sys.exit("Cannot find a version number in %s." % input_file)
 
-version_number = input_data[n + 5:n + 7]
+version_number = input_data[n + 5:n + 7].decode("ascii")
```

Comment:
Just confirming—this is encoded specifically in ascii and not utf-8?

Reply:
Yeah, I think they can actually be used interchangeably when decoding digits. The upstream code uses ascii, though, and it seems to work fine so far.